<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="loadedLayers">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="chooseLayerLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Choose Layer:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="convertToRDFButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert to RDF</string>
   </property>
  </widget>
  <widget class="QLabel" name="convertLayerLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Convert Layer to RDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="vocabularyCBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>GeoSPARQL Ontology</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>W3C Geo Vocabulary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Schema.org Vocabulary</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OSMRDF Vocabulary</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="chooseLayerLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vocabulary:</string>
   </property>
  </widget>
  <widget class="QLabel" name="chooseLayerLabel_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Literal Types:</string>
   </property>
  </widget>
  <widget class="QgsCheckableComboBox" name="literalTypeCBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>251</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>GeoJSON Literal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GML Literal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KML Literal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WKT Literal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WKB Literal</string>
    </property>
   </item>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCheckableComboBox</class>
   <extends>QComboBox</extends>
   <header>qgscheckablecombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
